src/test/kotlin/no/nav/syfo/api/SendPapirSykmeldingManuellOppgaveTest.kt

```diff
@@ -40,22 +40,7 @@ import no.nav.syfo.client.SyfoTilgangsKontrollClient
 import no.nav.syfo.client.Tilgang
 import no.nav.syfo.clients.KafkaProducers
 import no.nav.syfo.log
-import no.nav.syfo.model.AktivitetIkkeMulig
-import no.nav.syfo.model.Arbeidsgiver
-import no.nav.syfo.model.Diagnose
-import no.nav.syfo.model.HarArbeidsgiver
-import no.nav.syfo.model.IdentInfo
-import no.nav.syfo.model.IdentInfoResult
-import no.nav.syfo.model.MedisinskArsak
-import no.nav.syfo.model.MedisinskArsakType
-import no.nav.syfo.model.MedisinskVurdering
-import no.nav.syfo.model.OpprettOppgaveResponse
-import no.nav.syfo.model.PapirSmRegistering
-import no.nav.syfo.model.Periode
-import no.nav.syfo.model.Samhandler
-import no.nav.syfo.model.SmRegisteringManuellt
-import no.nav.syfo.model.Status
-import no.nav.syfo.model.ValidationResult
+import no.nav.syfo.model.*
```

Comment:
Så rart at den fortsatt skal ha stjerne for denne..?

Reply:
Fortsatt * her..?

Reply:
Nå! Burde det være i orden. EDB as.